crates/rooch-indexer/src/models/states.rs

```diff
@@ -131,3 +135,43 @@ impl StoredLeafState {
         Ok(state)
     }
 }
+
+#[derive(Clone, Debug, Queryable, Insertable, QueryableByName)]
+#[diesel(table_name = state_change_sets)]
+pub struct StoredStateChangeSet {
+    /// The tx order of this transaction which produce the state change set
+    #[diesel(sql_type = diesel::sql_types::BigInt)]
+    pub tx_order: i64,
+    /// The state change set, json format
+    #[diesel(sql_type = diesel::sql_types::Text)]
+    pub state_change_set: String,
```

Comment:
If we store the whole `ChangeSet` into a json, how do we provide `ChangeSet` filter base on table?

Reply:
Split StateChangeSet into TableChangeSet dimensions to store?

Reply:
Yes, we can keep the table handle in this table. Then, the developer can filter ChangeSet by table handle. The 0x0 is the global Object table.